packages/core-data/src/test/actions.js

```diff
@@ -30,6 +31,39 @@ describe( 'editEntityRecord', () => {
 	} );
 } );
 
+describe( 'deleteEntityRecord', () => {
+	it( 'triggers a DELETE request for an existing record', async () => {
+		const post = 10;
+		const entities = [
+			{ name: 'post', kind: 'postType', baseURL: '/wp/v2/posts' },
+		];
+		const fulfillment = deleteEntityRecord( 'postType', 'post', post );
+
+		// Trigger generator
+		fulfillment.next();
+
+		// Start
+		expect( fulfillment.next( entities ).value.type ).toBe(
+			'DELETE_ENTITY_RECORD_START'
+		);
+
+		// delete api call
+		const { value: apiFetchAction } = fulfillment.next();
+		expect( apiFetchAction.request ).toEqual( {
+			path: '/wp/v2/posts/10',
+			method: 'DELETE',
+		} );
+
+		expect( fulfillment.next().value.type ).toBe( 'REMOVE_ITEMS' );
+
+		expect( fulfillment.next().value.type ).toBe(
+			'DELETE_ENTITY_RECORD_FINISH'
+		);
+
+		expect( fulfillment.next().value ).not.toBe( false );
```

Comment:
Check for `.done`.

Reply:
Do you mean replace 
`expect( fulfillment.next().value ).not.toBe( false );`
with
`expect( fulfillment.next().done ).toBe( true );`
?

Reply:
`.toBe()` or `.not.toBe()`… that is the question

Reply:
I would check against the whole expected shape of `.next()`.